index.js

```diff
@@ -1,53 +1,47 @@
 'use strict';
 module.exports = function (str) {
-	var result = "";
-	var x, y, len = str.length;
+	var result = '';
+	var x;
+	var len = str.length;
 	var betweenQuotes = false;
 	var QuoteChar;
-	for(x = 0; x < len; x++) {
-		//FOUND DOUBLE-QUOTE OR SINGLE-QUOTE
-		if(str[x] === '"' || str[x] === '\'') {
-			//IF NOT PROCESSING IN BETWEEN QUOTES
-			if(!betweenQuotes) {
+	for (x = 0; x < len; x++) {
+		// FOUND DOUBLE-QUOTE OR SINGLE-QUOTE
+		if (str[x] === '"' || str[x] === '\'') {
+			// IF NOT PROCESSING IN BETWEEN QUOTES
+			if (betweenQuotes === false) {
```

Comment:
I had !betweenQuotes on the initial version but linter-xo pointed an error there when I ran the fix. Maybe the chain of issues below caused linter-xo to report that as an error as well?